gum/backend-linux/gumprocess-linux.c

```diff
@@ -605,7 +620,7 @@ gum_do_modify_thread (gpointer data)
 beach:
   {
     if (attached)
-      gum_libc_ptrace (PTRACE_DETACH, ctx->thread_id, NULL, NULL);
+      gum_libc_ptrace (PTRACE_DETACH, ctx->thread_id, NULL, (void *) SIGCONT);
```

Comment:
```suggestion
      gum_libc_ptrace (PTRACE_DETACH, ctx->thread_id, NULL, (void *) SIGCONT);
```

Should always use GLib's casts for `void *` casts, e.g. `GINT_TO_POINTER()`.

Reply:
Done.